README.md

```diff
@@ -140,43 +152,43 @@ features = ["from", "add", "iterator"]
 
 And this to the top of your Rust file:
 ```rust
-extern crate derive_more;
 // use the derives that you want in the file
 use derive_more::{Add, Display, From};
 ```
 If you're still using Rust 2015, add this instead:
-```rust
+```rust,edition2015
 extern crate core;
 #[macro_use]
 extern crate derive_more;
+# fn main() {} // omit wrapping statements above into `main()` in tests
```

Comment:
This line is not hidden in github its rendering of the readme. Which is the main reason I copied lib.rs to the readme all the time. I don't think there's a way around this sadly.

Reply:
@JelteF yes, I've tried but haven't found a way around it. Still, I think having this line exposed in GitHub's README is a lesser evil than duplication and copy-squating.